README.md

```diff
@@ -0,0 +1,155 @@
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h1><b>Blog app</b></h1>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+## 📗 Table of Contents
+
+- [📖 Blog-app ](#-Create a Blog-app in RubyonRails )
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [🚀 Presentation Video](#Video)
+    - [Install](#install)
+    - [Usage](#usage)
+  - [👥 Author ](#-author-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Blog app <a name="about-project"></a>
+
+**Blog App** 
+Blog-app is a fully functioning website that show the list of posts and empower readers to interact with them by adding comments and liking posts.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Create Database named Blog_app_development**
+- **Create tables by migration files**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```
+    ruby
+    rails
+    postgresql
+    bundler
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```bash
+  git clone https://github.com/cosywasswa/Blog-app.git
+```
+
+### Install
+
+Install this project with:
+
+```bash
+  cd Blog-app
+  bundle install
+  .stylelintrc.json
+  .github/workflows/linters.yml
+```
+
+it will install the required gemfile for running the project
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="author"></a>
+
+👤 **Cosywasswa**
+
+- GitHub: [@cosywasswa](https://github.com/cosywasswa)
+- LinkedIn: [cosmas-wasswa](https://www.linkedin.com/in/cosmas-wasswa)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+ **Validations and Model specs.**
+ **Processing data in models.**
+ **Controllers.**
+ **Controllers specs.**
+ **Views.**
+ **Forms.**
+ **Integration specs for Views and fixing n+1 problems.**
+ **Add Devise.**
+ **Add authorization rules.**
+ **Add API endpoints.**
+ **API documentation.**
```

Comment:
- [ ] [Optional][Highly recommended] It would be better professionally if you make all feature features as bullet points or checklist because now it's listed beside each other see the image below.

![image](https://github.com/cosywasswa/Blog-app/assets/68796232/c3fcfc1b-433d-4621-9f03-a0162084e4fa)

Reply:
ok thanks for feedback